### ISSUE TITLE
Use of undeclared type URLRequest, when deploying to Heroku

### ISSUE THREAD

**Issue by nickfarrant** [pusher/push-notifications-server-swift#18]
Hello,

I am using this library with Vapor 3, and all works fine until deployment, where I get the following error:

```
[24/26] Compiling PushNotifications JWTTokenGenerable.swift
/tmp/build_ad5fcca8/.build/checkouts/push-notifications-server-swift/Sources/PushNotifications/PushNotifications.swift:411:42: error: use of undeclared type 'URLRequest'
    private func networkRequest(request: URLRequest, completion: @escaping (_ result: Result<Data, Error>) -> Void) {
                                         ^~~~~~~~~~
/tmp/build_ad5fcca8/.build/checkouts/push-notifications-server-swift/Sources/PushNotifications/PushNotifications.swift:434:81: error: use of undeclared type 'URLRequest'
    private func setRequest(url: URL, httpMethod: String, body: Data? = nil) -> URLRequest {
                                                                                ^~~~~~~~~~
/tmp/build_ad5fcca8/.build/checkouts/push-notifications-server-swift/Sources/PushNotifications/PushNotifications.swift:412:36: error: 'URLSessionConfiguration' is unavailable: This type has moved to the FoundationNetworking module. Import that module to use it.
        let sessionConfiguration = URLSessionConfiguration.default
                                   ^~~~~~~~~~~~~~~~~~~~~~~
Foundation.URLSessionConfiguration:2:18: note: 'URLSessionConfiguration' has been explicitly marked unavailable here
public typealias URLSessionConfiguration = AnyObject
                 ^
/tmp/build_ad5fcca8/.build/checkouts/push-notifications-server-swift/Sources/PushNotifications/PushNotifications.swift:413:23: error: 'URLSession' is unavailable: This type has moved to the FoundationNetworking module. Import that module to use it.
        let session = URLSession.init(configuration: sessionConfiguration)
                      ^~~~~~~~~~
Foundation.URLSession:2:18: note: 'URLSession' has been explicitly marked unavailable here
public typealias URLSession = AnyObject
                 ^
/tmp/build_ad5fcca8/.build/checkouts/push-notifications-server-swift/Sources/PushNotifications/PushNotifications.swift:435:23: error: use of unresolved identifier 'URLRequest'
        var request = URLRequest(url: url)
                      ^~~~~~~~~~
 !     Push rejected, failed to compile Swift app.
 !     Push failed
```

Is this a known issue at all?

Thanks

**Comment by Xaandeer** [comment]
I had the same issue while compiling in a docker container. It can be solved by just adding
```swift
#if canImport(FoundationNetworking)
import FoundationNetworking
#endif
```
to the `PushNotifications.swift` file.

You can find a more detailed explanation why this is necessary here: [https://stackoverflow.com/a/58606520](https://stackoverflow.com/a/58606520)

The solution is already implemented in the pull request #17, in the meantime I would recommend you to fork the repo, add the two lines of code by yourself and use your own version until the problem is fixed.

**Comment by danielrbrowne** [comment]
@nickfarrant @Xaandeer Apologies in the large delay in reviewing this PR. This fix is now released as of v1.0.3
